ui/src/main/webapp/src/app/reports/filter/report-filter.component.html

```diff
@@ -2,7 +2,7 @@
 
 <div *ngIf="!isFilterUpToDate" class="alert alert-warning alert-dismissable">
     <span class="pficon pficon-warning-triangle-o"></span>
-    <strong>Info: </strong> Applications displayed in filter are corresponding to last execution state.
+    Applications displayed in filter are corresponding to last execution state.
```

Comment:
@jsight Ok then how do you understand this:

> (GENERAL FOR ALL INFO/ERROR MESSAGES)
Remove the “Info: ” or “Error: ” text before the message. The icon and color are enough to signalize the nature of the message.